### PR TITLE
Rails deploy

### PR DESCRIPTION
**Pull request by neo-tahi**
#REFERENCES:
https://www.pivotaltracker.com/epic/show/1595196

This pull request introduces a push-style deployment strategy using ansible. Currently, you must run the ansible commands from your local workstation, but we're planning to add an additional server instance to play the role of ansible controller.

We're running on top of the [debops](http://debops.org/) framework, a set of opinionated ansible playbooks that all are working in harmony to make a complete server. There is a known and discussed constraint that these servers will only be deployed in a debian-based server setup.

See the readme for instructions on how to run the commands introduced.

### CODE REVIEW

**Comment by neo-tahi** [comment]
@DTrejo and @afomi Worked on this as well, but I'm not sure how to properly attribute them, since we didn't make any commits (aside from WIP commits). Shoot me a suggestion if you have one.

File: ansible/inventory/group_vars/all/rails_deploy.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+# See http://docs.debops.org/en/latest/ansible/roles/debops.rails_deploy.html#high-level-goals
+# For explanation and more defaults.
+
+ferm: False
+
+ruby_version: "ruby-2.1.5"
```
Comment (mikem) [inline ansible/inventory/group_vars/all/rails_deploy.yml]: Can things like this be stored in a config file? Or will we have to update this file (and possibly others) when we want to bump the Ruby version?

Comment (mikem) [inline ansible/inventory/group_vars/all/rails_deploy.yml]: I realize this might be a config file :-)

**Comment by mikem** [comment]
Looks OK to me. :+1:

File: ansible/inventory/group_vars/all/rails_deploy.yml (hunk 1 above)
Comment (neo-tahi) [inline ansible/inventory/group_vars/all/rails_deploy.yml]: Yeah, when we update ruby versions, there's yet another place to update it. But it beats manually updating all of the servers. :wink:

Comment (mikem) [inline ansible/inventory/group_vars/all/rails_deploy.yml]: OK cool, as long as it's just one place for all the Ansible playbooks.

**Pull request merged**